src/main/scala/cromwell/util/google/GoogleCloudStorage.scala

```diff
@@ -36,6 +36,11 @@ case class GoogleCloudStorage(client: Storage) {
     new GcsBucketInfo(bucketName, bucket.getLocation, bucket.getTimeCreated, bucket.getOwner)
   }
 
+  def getCrc32c(googleCloudStoragePath: GoogleCloudStoragePath): String = {
```

Comment:
Could you add a comment that addresses @geoffjentry's question, maybe with a link to the relevant GCS docs?